spec/features/companies/user_creates_a_new_contact_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+describe 'user creates a contact' do
+  scenario 'user creates a contact and sees it on company page' do
+    company = Company.create(name: "ESPN", city: "LA")
+    cat1 = Category.create(title: "News")
+    job = company.jobs.create(title: "Journalist", level_of_interest: 70, category: cat1, description: "Write about sports events")
```

Comment:
i like how you're creating the job based on the company here and leveraging your associations 👍